data/sql_updates/totals_candidate_committee_presidential.sql

```diff
@@ -188,12 +188,14 @@ with last_cycle as (
             true as full_election
         from
             cycle_totals totals
-            left join first_election first using (candidate_id, election_year)
-            left join last_election last using (candidate_id, election_year)
+            left join ofec_candidate_election_mv_tmp election on
```

Comment:
I think we need the same thing in data/sql_updates/totals_candidate_committee_house_senate.sql

This will need an update to webservices/flow.py and that should fix the test